sidekick_core/lib/src/entrypoint.dart

```diff
@@ -0,0 +1,9 @@
+import 'dart:io';
+
+class EntryPoint {
```

Comment:
Since this class just wraps a `File` and has no other functionality, should we remove it?